src/components/KeyboardShortcuts/index.spec.tsx

```diff
@@ -167,7 +171,11 @@ describe(__filename, () => {
   });
 
   it('renders one line descriptions for keys "p"/"n" and "k"/"j" in Browse', () => {
-    const root = render({ compareInfo: null });
+    const root = render({
+      file: createInternalVersionFile(
+        fakeVersionFileWithContent,
+      ) as VersionFileWithContent,
```

Comment:
So, does that mean TS isn't able to infer the type of the returned value here?

Reply:
I'm pretty sure that is the case, or I wouldn't have done it, but I'll double check.

Reply:
It turns out it's not actually needed here, but that's not because TS is able to infer the type, it's just because there is no code below this that relies on it being a particular type. There are plenty of other places where we do need to assign an explicit type to the result of `createInternalVersionFile` even though one might expect TS to be able to infer it.

I have removed these extra `as`'s where we don't actually need them.